app/controllers/account/benefits/enrollment/elect.js

```diff
@@ -116,5 +117,27 @@ export default class AccountBenefitsEnrollmentElectController extends Controller
         });
       }
     }
+    this.coverageChangeOccured = true;
+  }
+
+  @action
+  update () {
+    //check if the model is truly dirty
+    if (this.model.enrollment.hasDirtyAttributes || this.coverageChangeOccured) {
+      if (this.updating) {
+        //if we already have a runloop set up, return;
+        return;
+      }
+      this.updating = true;
+
+      //save record in 10 seconds
+      run.later(() => {
+        if (!this.isDestroyed && !this.isDestroying) {
+          this.data.saveRecord(this.model.enrollment);
+          this.updating = false;
+          this.coverageChange = false;
```

Comment:
Is this supposed to be
```suggestion
          this.coverageChangeOccured = false;
```